Install_for_Linux.md

```diff
@@ -0,0 +1,11 @@
+How to Install Vital on Ubuntu 
```

Comment:
Cool Pull Request! Here is some feedback:

* Instead of making a new file, you would probably want to edit the README.md, in the `Installing` section.
* You generally wouldn't want to include instructions this verbose regarding what buttons to click etc. because websites change often and trying to maintain a README like that would be difficult.
* A good example of OS-specific install instructions would be parts that were unique to that OS. Much of these instructions are the same regardless of which OS the user is using.

(btw only offering this feedback because this is generally part of the GitHub process; you open PRs and get feedback, but if I was a maintainer of this project, I would not merge this... maybe except for the period-additions in the README.md, just cuz).

good luck out there! <3